### PR TITLE
amp-form-textarea text area drag fix.

### PR DESCRIPTION
**Pull request by alabiaga**
call measure mutate every time textarea mousedown event is triggered to notify viewer of possible doc height change.

b122451407

### CODE REVIEW

**Comment by alabiaga** [comment]
as discussed with @cvializ offline, there should be some way for runtime to correct these issues. It is the responsibility of the developer creating the component to ensure that any DOM changes should be propagated to the viewer via measureMutateElement but runtime should still have a check that runs less frequently than doPass to verify that any discrepancy between the cached and new doc height be resolved and updated.

File: extensions/amp-form/0.1/amp-form-textarea.js [hunk 1]
```diff
@@ -101,8 +101,7 @@ export class AmpFormTextarea {
       }
 
       const element = dev().assertElement(e.target);
-      if (element.tagName != 'TEXTAREA' ||
-          !element.hasAttribute(AMP_FORM_TEXTAREA_EXPAND_ATTR)) {
+      if (element.tagName != 'TEXTAREA') {
```
Comment (cvializ) [inline extensions/amp-form/0.1/amp-form-textarea.js]: Can we add a comment stating why we handle all `textarea` elements and not only ones with the feature attribute?

**Comment by alabiaga** [comment]
Thanks Carlos.

**Comment by cvializ** [comment]
np!

File: extensions/amp-form/0.1/amp-form-textarea.js (hunk 1 above)
Comment (alabiaga) [reply extensions/amp-form/0.1/amp-form-textarea.js]: apologies for missing this comment. I'll make the comment change in a separate PR, thanks.

**Pull request merged**